style/style.css

```diff
@@ -632,3 +727,449 @@ form .line {
   left: 120px;
   margin-bottom: 8px;
 }
+
+@media (min-width: 768px) {
+  body {
+    width: 100%;
+    box-sizing: border-box;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  nav {
+    margin-top: -15px;
+  }
+
+  ul {
+    display: flex;
+    align-items: center;
+    float: right;
+    gap: 40px;
+    margin-right: 143px;
+    margin-top: -47px;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  li a {
+    width: 63px;
+    height: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+    margin-top: -15px;
+  }
+
+  .info {
+    background: url("../style/img/desktop/Header\ bg@2x.png");
+    background-size: cover;
+    width: 1440px;
+    margin-bottom: 180px;
+    position: relative;
+    left: 500px;
+    top: 100px;
+  }
+
```

Comment:
- I think the contents in your portfolio's header are a bit too left-aligned. Can you kindly fix that?
- Here is what I can see compared to what is on figma 👇🏽 
### Yours
![exessive](https://user-images.githubusercontent.com/99246359/200008607-1e2c11b9-e15a-4fde-a4d8-e055f15e5d26.PNG)

### Figma

![fighead](https://user-images.githubusercontent.com/99246359/200008749-051dcefb-5618-4b74-8250-be0cf3d323b0.PNG)

- I think it will be better if you can kindly make the spacing almost identical to what's on Figma. Perhaps, you can as well start by reducing the `left: 500px;` on line 779 to something lesser. Maybe `left: 300px;`, then checkout for any excess margin-left or padding-left that has been assigned to the text. 🍷